test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -135,7 +138,7 @@ const loopbackBiometricsErrorMobileMock = RequestMock()
   .onRequestTo(/\/idp\/idx\/authenticators\/poll/)
   .respond((req, res) => {
     res.headers['content-type'] = 'application/json';
-    if (probeSuccess) {
```

Comment:
Using one variable `probeSuccess` for several tests using same mock is definitely a source of flackiness in concurrent mode